api.planx.uk/modules/file/service/getFile.ts

```diff
@@ -1,25 +1,25 @@
-import type S3 from "aws-sdk/clients/s3.js";
+import type { PutObjectCommandInput } from "@aws-sdk/client-s3";
 import { s3Factory } from "./utils.js";
 
 export const getFileFromS3 = async (fileId: string) => {
   const s3 = s3Factory();
 
   const params = {
     Key: fileId,
-  } as S3.PutObjectRequest;
+  } as PutObjectCommandInput;
 
-  const file = await s3.getObject(params).promise();
+  const file = await s3.getObject(params);
 
   return {
-    body: file.Body as Buffer,
+    body: file.Body,
```

Comment:
Removing this causes type issues in `modules/file/controller` because it expects the body to be a `Buffer` - ```Argument of type 'StreamingBlobPayloadOutputTypes | undefined' is not assignable to parameter of type 'Buffer | undefined'. ```